src/app/pages/HomePage/ParaTimeSelector/index.tsx

```diff
@@ -0,0 +1,123 @@
+  disabled: boolean
+}
+
+const ParaTimeSelectorGlow = styled(Box, {
+  shouldForwardProp: (prop: string) =>
+    !(['disabled'] as (keyof ParaTimeSelectorProps)[]).includes(prop as keyof ParaTimeSelectorProps),
+})<ParaTimeSelectorProps>(({ disabled }) => ({
+  position: 'relative',
+  width: '80vh',
+  height: '80vh',
+  marginTop: '-17vh',
+  backgroundImage: `url("${paratimeSelectorGlow}")`,
+  backgroundSize: 'contain',
+  backgroundPosition: 'center',
+  backgroundRepeat: 'no-repeat',
+  ...(disabled
+    ? {
+        opacity: 0.25,
+      }
+    : {}),
+}))
+
+const ParaTimeSelectorGlobe = styled(Box)(() => ({
+  position: 'absolute',
+  width: '65%',
+  paddingBottom: '65%',
+  left: '50%',
+  bottom: '8%',
+  transform: 'translateX(-50%)',
+  backgroundImage: `url("${paratimeSelectorGlobe}")`,
+  backgroundSize: 'cover',
+  backgroundPosition: 'center',
+  backgroundRepeat: 'no-repeat',
+  overflow: 'hidden',
+  borderRadius: '50%',
+}))
+
+const ExploreBtn = styled(Button)(({ theme }) => ({
+  position: 'absolute',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  paddingLeft: theme.spacing(6),
+  paddingRight: theme.spacing(6),
+  width: 'max-content',
+}))
+
+const ZoomOutBtn = styled(Button)(({ theme }) => ({
+  position: 'absolute',
+  top: theme.spacing(4),
+  left: '50%',
+  transform: 'translateX(-50%)',
+}))
+
+const ZoomOutBtnFade = styled(Fade)(() => ({
+  transitionDelay: '500ms !important',
+}))
+
+const ParaTimeSelectorCmp: FC<ParaTimeSelectorProps> = ({ disabled }) => {
+  const { t } = useTranslation()
+  const [step, setStep] = useState<ParaTimeSelectorStep>(ParaTimeSelectorStep.ENABLE_EXPLORE)
+  const [selectedGraphEndpoint, setSelectedGraphEndpoint] = useState<GraphEndpoint>(GraphEndpoint.CONSENSUS)
+
+  const onExploreClick = () => {
+    setStep(ParaTimeSelectorStep.EXPLORE)
+  }
+
+  const onZoomOutClick = () => {
+    setSelectedGraphEndpoint(GraphEndpoint.CONSENSUS)
+  }
+
+  return (
+    <ParaTimeSelectorGlow disabled={disabled}>
+      <ParaTimeSelectorGlobe>
+        <Graph
+          disabled={disabled}
+          transparent={ParaTimeSelectorUtils.getIsGraphTransparent(step)}
+          selectedGraphEndpoint={selectedGraphEndpoint}
+          setSelectedGraphEndpoint={setSelectedGraphEndpoint}
+        />
+        <ZoomOutBtnFade in={ParaTimeSelectorUtils.showZoomOutBtn(selectedGraphEndpoint)}>
+          <ZoomOutBtn
+            variant="text"
+            color="secondary"
+            startIcon={<ChevronIcon arrowDirection={ArrowDirection.LEFT} />}
+            onClick={onZoomOutClick}
+            disabled={disabled}
+          >
+            {t('home.zoomOutBtnText')}
+          </ZoomOutBtn>
+        </ZoomOutBtnFade>
+        {ParaTimeSelectorUtils.showExploreBtn(step) && (
+          <ExploreBtn
+            color="secondary"
+            variant="contained"
+            onClick={onExploreClick}
+            aria-label={t('home.exploreBtnText') ?? undefined}
```

Comment:
remove undef?

Reply:
I don't know how this is possible:
```tsx
  const a1 = t('home.exploreBtnText')
    // hover:
    const t: TFunction <"home.exploreBtnText", string, StringMap>(
      key: "home.exploreBtnText" | "home.exploreBtnText"[]
    ) => string (+11 overloads)
```
```tsx
  const b1 = <div>{t('home.exploreBtnText')}</div>
    // hover:
    const t: TFunction<"home.exploreBtnText", DefaultTFuncReturn, StringMap>(
      key: "home.exploreBtnText" | "home.exploreBtnText"[]
    ) => DefaultTFuncReturn (+11 overloads)
```

This isn't nullable
```tsx
  const a1 = t('home.exploreBtnText')
  const a2 = <div aria-label={a1}></div>
    // hover:
    const a1: string
```

Even this isn't nullable
```tsx
  const c1 = <>{t('home.exploreBtnText')}</>
    // hover:
    const t: TFunction<"home.exploreBtnText", string, StringMap>(
      key: "home.exploreBtnText" | "home.exploreBtnText"[]
    ) => string (+11 overloads)
```

Reply:
The type it defaults to is 
```
export type DefaultTFuncReturn = string | (TypeOptions['returnNull'] extends true ? null : never);
```

But you are right, not defining it inline, infers the correct type.